web-client/src/app/views/two-factor-authentication/two-factor-authentication.page.ts

```diff
@@ -69,6 +76,12 @@ export class TwoFactorAuthenticationPage implements OnInit {
     }
   }
 
+  ngOnDestroy() {
```

Comment:
You don't have to use `ngOnDestroy`, we have `UntilDestroyed` utility for unsubscribe automatically for us